content/articles/understanding-blockchains-interoperability-via-cosmos-network/index.md

```diff
@@ -0,0 +1,94 @@
+
+The messages conveyed between the networking layers contain various data ranging from blocks, proposals, and even consensus votes. The [validator nodes](https://learn.radixdlt.com/article/what-is-a-validator-node) work alongside the non-validator nodes. This allows efficient interaction between the nodes, and network peers can quickly access the latest messages and transactions.
+
+#### The application layer
+As discussed earlier, the work of the consensus layer is to perform nodes validation. In contrast, the networking layer transmits messages. The application layer then passes transactions to networking and consensus layers.
+
+In this layer, the transactions happening on the blockchain are determined. They are then forwarded to the consensus layer. The consensus and application layer communication happen through an [Application Blockchain Interface](https://blog.cosmos.network/abci-the-application-blockchain-interface-f1bd8278cdd7) (ABCI). This interface is not limited to a specific programming language. Applications developed to operate on the cosmos network can interact with the base layers of the protocol. This is with disregard of the programming language used.
+
+### Pillars of the cosmos network
+#### Tendermint core
+Since the cosmos network utilizes different technological innovations, the efficiency and security of the interoperable systems are assured. Some of these technological innovations include the [Tendermint core](https://tendermint.com/core/).
+
+The Tendermint core and other vital components form an important building block of the cosmos hub in the cosmos network. It includes the [Byzantine Fault Tolerance](https://www.fool.com/investing/stock-market/market-sectors/financials/cryptocurrency-stocks/byzantine-fault-tolerance) feature in its network architecture. The feature handles the regeneration of applications using the existing blockchain networks in the cosmos environment. This means it assists the users to switch to other existing blockchain networks.
+
+For instance, an ABCI application can assist in deploying the Ethereum codebases on the cosmos network. This will ensure that the challenges of transferring [EVM](https://ethereum.org/en/developers/docs/evm/) to the cosmos are dealt with.
+
+#### IBC protocol
+This [protocol](https://ibcprotocol.org/) handles interoperability between different blockchains in the cosmos network. It also sets up the interaction between blockchains through the [zone and hub model](https://www.preethikasireddy.com/post/how-does-cosmos-work-how-does-it-compare-to-bitcoin-and-ethereum-part-2). Hubs are used to manage different zones. There is indirect interaction between the zones through the use of IBC. The reason for indirect interaction is to ensure that the scalability of the blockchain is not affected.
```

Comment:
The [Inter‑Blockchain Communication Protocol (IBC)](https://ibcprotocol.org/) handles interoperability between different blockchains in the cosmos network. It also sets up the interaction between blockchains through the [zone and hub model](https://www.preethikasireddy.com/post/how-does-cosmos-work-how-does-it-compare-to-bitcoin-and-ethereum-part-2). Hubs are used to manage different zones. There is indirect interaction between the zones through the use of IBC. The reason for indirect interaction is to ensure that the scalability of the blockchain is not affected.